packages/flutter/lib/src/rendering/view.dart

```diff
@@ -427,3 +442,37 @@ class RenderView extends RenderObject with RenderObjectWithChildMixin<RenderBox>
 /// Used by [RenderView.debugAddPaintCallback] and
 /// [RenderView.debugRemovePaintCallback].
 typedef DebugPaintCallback = void Function(PaintingContext context, Offset offset, RenderView renderView);
+
+/// Interface for a manager of [RenderView]s to produce frames for the
```

Comment:
If it's a manager, it could be called `RenderViewManager`. The definition of "repository" is "a central location in which data is stored and managed", which implies ownership of the data.  Does it also take ownership of the views?

Reply:
The RenderViewRepository crucially does not own the views. It just manages them for the purposes of frame production. I guess, RenderViewManager is a better name for it, although I dislike the term "manager"... I'll change it until someone has a better idea :)